components/organisms/MujeresDeOracion/MujeresDeOracion.module.css

```diff
@@ -0,0 +1,150 @@
+.MujeresDeOracion {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.MujeresDeOracion_welcome {
+  margin-top: 6.25rem;
+  width: 100%;
+  height: 80vh;
+  object-fit: cover;
+  object-position: 50% 50%;
+  background-image: linear-gradient(
+      rgba(255, 97, 97, 0.716),
+      rgba(255, 97, 97, 0.716)
+    ),
+    url("/gruposDesafiantes/mujeresDeOracion/mujeresdeoracion.webp");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 0.625rem;
+}
+
+.MujeresDeOracion-title_welcome {
+  color: var(--white);
+  font-size: var(--fontSize-superTitles_mobile);
+  text-align: center;
+}
+
+.MujeresDeOracion-paragraph_welcome {
+  color: var(--white);
+  font-size: var(--fontSize-paragraph_mobile);
+  text-align: center;
+  width: 15.625rem;
+}
+
+.MujeresDeOracion-buttons {
+  width: 100%;
+  height: 30vh;
+  margin-top: 1.25rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 0.625rem;
+}
+
+.MujeresDeOracion-buttons_bgTrasnsparent {
+  border: var(--greenDesafiante) solid 0.02rem;
+  border-radius: var(--borderRadius);
+  font-size: var(--fontSize-buttons_mobile);
+  background: var(--white);
+}
+
+.MujeresDeOracion-buttons_bgGreen {
+  border: var(--greenDesafiante) solid 0.02rem;
+  border-radius: var(--borderRadius);
+  font-size: var(--fontSize-buttons_mobile);
+  background: var(--greenDesafiante);
+  color: var(--white);
+}
+
+.MujeresDeOracion-content_postsShedule {
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 1.25rem;
+  margin-top: 1.875rem;
+  margin-bottom: 1.875rem;
+}
+
+.MujeresDeOracion-image_postsShedule {
+  border-radius: var(--borderRadius);
+  width: 100%;
+}
+
+@media (width >= 48rem) {
+  .MujeresDeOracion-title_welcome {
+    font-size: var(--fontSize-superTitles_tablet);
+  }
+
+  .MujeresDeOracion-paragraph_welcome {
+    font-size: var(--fontSize-paragraph_tablet);
+  }
+
+  .MujeresDeOracion-buttons_bgTrasnsparent,
+  .MujeresDeOracion-buttons_bgGreen {
+    font-size: var(--fontSize-buttons_tablet);
+  }
+}
+
+@media (width >=64rem) {
```

Comment:
![image](https://github.com/generaciondesafiante/Regresemos_a_casa_UI/assets/104586980/b68224d5-3e9d-4c13-9321-d99c9407b047)
A partir de esta medida los botones van horizontal

Reply:
added the button border